semeio/jobs/design2params/design2params.py

```diff
@@ -0,0 +1,166 @@
+import logging
+import sys
+
+import numpy as np
+import pandas as pd
+
+import logging
+
+
+# Filenames created/modified by this script. Don't change unless you understand the consequences.
+_TARGET_FILE_TXT = "DESIGN2PARAMS.OK"
+_PARAMETERS_TXT = "parameters.txt"
```

Comment:
This requires that we run the job in the same folder as `parameters.txt`, will that always be the case? Would be nice to do the parsing of `parameters.txt` before this job to make it more independent of the file system. Could add it as an optional argument with a default value.